config/superblocks.ts

```diff
@@ -32,13 +32,13 @@ export enum SuperBlocks {
  * 'Upcoming' is for development -> not shown on stag or prod anywhere
  */
 export enum SuperBlockStages {
-  FrontEnd = 'frontend',
-  Backend = 'backend',
-  Python = 'python',
-  Extra = 'extra',
-  Legacy = 'legacy',
-  New = 'new',
-  Upcoming = 'upcoming'
+  FrontEnd = 'Front End Development',
+  Backend = 'Back End Development',
+  Python = 'Python & AI',
+  Extra = 'Extra',
+  Legacy = 'Legacy',
+  New = 'New',
+  Upcoming = 'Upcoming'
```

Comment:
This should never have had string values. I've created https://github.com/freeCodeCamp/freeCodeCamp/pull/51178 to fix that, but please revert this change. It should be fine to use the enums directly.

Reply:
Alrighty.